multinode-demo/fullnode.sh

```diff
@@ -488,7 +488,9 @@ while true; do
         new_state_archive="$SOLANA_RSYNC_CONFIG_DIR"/new_state.tgz
         (
           rm -rf "$new_state_dir" "$new_state_archive"
-          cp -a "$state_dir" "$new_state_dir"
+          mkdir -p "$new_state_dir"
+          cp -a "$state_dir"/snapshots "$new_state_dir"
```

Comment:
doh I totally broke this in 465cd45833b6f9664f7fdbfeabb8e4758dfc4392.

Can you please add a big comment here that explains why snapshots must be copied before accounts to prevent another attempt to consolidate these two `cp` commands in the future